README.md

```diff
@@ -123,27 +102,61 @@ models:
 
 > Note: If your profile does not have permissions to create schemas in your destination, you can set each `+schema` to blank. The package will then write all tables to your pre-existing target schema.
 
-## Contributions
+### Deprecated `CAMPAIGN_SUPRESSION_LIST_HISTORY` table
+
+The Iterable connector schema originally misspelled the `CAMPAIGN_SUPPRESSION_LIST_HISTORY` table as `CAMPAIGN_SUPRESSION_LIST_HISTORY` (note the singular `P`). As of August 2021, Fivetran has deprecated the misspelled table and will only continue syncing the correctly named `CAMPAIGN_SUPPRESSION_LIST_HISTORY` table.
+
+By default, this package refers to the new table (`CAMPAIGN_SUPPRESSION_LIST_HISTORY`). To change this so that the package works with the old misspelled source table (we do not recommend this, however), add the following configuration to your `dbt_project.yml` file:
+
+```yml
+vars:
+    iterable_source:
+        iterable__campaign_suppression_list_history_table: "campaign_supression_list_history"
+```
+### Change the source table references
+If an individual source table has a different name than what the package expects, add the table name as it appears in your destination to the respective variable:
+> IMPORTANT: See this project's [`dbt_project.yml`](https://github.com/fivetran/dbt_iterable_source/blob/main/dbt_project.yml) variable declarations to see the expected names.
```

Comment:
```suggestion
> IMPORTANT: See this project's [`dbt_project.yml`](https://github.com/fivetran/dbt_iterable/blob/main/dbt_project.yml) variable declarations to see the expected names.
```

Reply:
For this one I believe we want to keep the source yml since we want to show the source table names.

Reply:
sounds good!

Reply:
ah you are correct.